packages/schemas/src/gen/utils.test.ts

```diff
@@ -0,0 +1,24 @@
+import { getStringMaxLength, splitAtCommasOutsideParentheses } from './utils';
+
+test('splitAtCommasOutsideParentheses should split at each comma where are not surrounded by the parentheses', () => {
```

Comment:
consider fix the language: "where are not"

Reply:
where are => that is
the parentheses => a pair of parentheses

Reply:
Improved. PTAL

`splitAtCommasOutsideParentheses should split at each comma that is not surrounded by a pair of parentheses`